content/applications/general/email_communication/email_servers.rst

```diff
@@ -129,94 +129,80 @@ How to manage inbound messages
+   respective configuration view. To edit catchall and bounce aliases, you first need to activate
+   the :ref:`developer mode <developer-mode>`. Then go to :menuselection:`Settings --> Technical
+   --> Parameters --> System Parameters` to customize the aliases (*mail.catchall.alias* &
+   *mail.bounce.alias*).
 
- .. image:: email_servers/system-parameters.png
-    :align: center
+   .. image:: email_servers/system-parameters.png
+      :align: center
+      :alt: System parameters with catchall configuration in Odoo.
 
-.. note:: By default inbound messages are fetched every 5 minutes in Odoo on-premise.
-   You can change this value in :ref:`developer mode <developer-mode>`.
-   Go to :menuselection:`Settings --> Technical --> Automation -->
-   Scheduled Actions` and look for *Mail: Fetchmail Service*.
-    
+.. note:: By default inbound messages are fetched every 5 minutes in Odoo on-premise. You can
+   change this value in :ref:`developer mode <developer-mode>`. Go to :menuselection:`Settings -->
+   Technical --> Automation --> Scheduled Actions` and look for *Mail: Fetchmail Service*.
 
 Set up different dedicated servers for transactional and mass mails
 ===================================================================
 
-Odoo is subject to a :ref:`daily email limit <email_communication/daily_limit_mail>` to prevent abuse. 
-However, if needed, you can use a separate Mail Transfer Agent (MTA) servers for transactional
-e-mails and mass mailings.
-Example: use Odoo's own mail server for transactional e-mails, and Sendgrid, Amazon SES, or Mailgun
-for mass mailings. Another alternative is to use Postmark for transactional e-mails, and Amazon SES
-or Sendgrid for mass mailings.
+In Odoo a separate Mail Transfer Agent (MTA) server can be used for transactional e-mails and mass
+mailings. Example: Use Postmark or SendinBlue for transactional e-mails, and Amazon SES, Mailgun or
+Sendgrid for mass mailings.
 
 .. note::
-   A default outgoing email server is already configured. You should not create an alternative one
-   unless you want to use a specific external outgoing email server for technical reasons.
+   A default outgoing email server is already configured. Do not create an alternative one unless a
+   specific external outgoing email server is needed for technical reasons.
 
-To do this, you should first activate the :ref:`developer mode <developer-mode>` and then go to
-:menuselection:`Settings --> Technical --> Outgoing` e-mail servers. There you have to create two
-e-mail MTA server settings. One for transactional e-mails and one for mass mail servers. Be sure to
-mark the priority of transactional e-mail servers as low as the mass email servers.
+To do this, first activate the :ref:`developer mode <developer-mode>`, and then go to
+:menuselection:`Settings --> Technical --> Outgoing` e-mail servers. There, create two e-mail MTA
+server settings; one for the transactional e-mails and one for the mass mailing server. Make sure
```

Comment:
Replace `e-mails` with `emails` for consistency with the rest of the doc

(two instances of `e-mails` here)